stats/container_stats.go

```diff
@@ -141,15 +141,17 @@ func (s *ContainerStatsHandler) Handle(key string, initialMessage string, incomi
 		IDList := []string{}
 		bufioReaders := []*bufio.Reader{}
 		for i := 0; i < len(contList); i++ {
```

Comment:
Is there a  good reason for not ranging over the contList, ie `for _, container := range contList { ...} `.
If not, can you switch the for loop to use range? I think it will make this code easier to understand.